ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-rollback-nightly.yaml

```diff
@@ -62,6 +62,11 @@ releases:
       architecture: amd64
       channel: candidate
       version: 4.16.0
+  source:
```

Comment:
The rollback nightly ci does not reference openshift-extended-upgrade-pre, so we do not need the setting here.

Reply:
it has openshift-upgrade-qe-test-hypershift-rollback which has pre step, so keep it.